src/qml/ui/LayerTreeItemProperties.ui.qml

```diff
@@ -5,16 +5,22 @@ import QtQuick.Layouts 1.3
 Popup {
     property alias itemVisible: itemVisibleCheckBox.checked
 
+    leftPadding: 20 * dp
+    rightPadding: 20 * dp
+    topPadding: 20 * dp
+    bottomPadding: 10 * dp
+
     Page {
         header: Label {
             text: title
-            font.pointSize: 14
+            font.bold: true
+            font.pointSize: 18 * dp
         }
 
         CheckBox {
             id: itemVisibleCheckBox
             text: qsTr("Show on map canvas")
-            font.pointSize: 14
+            font.pointSize: 18 * dp
```

Comment:
@m-kuhn , re pointSize, this checkbox had a pointSize of 14 (without the dp multiplier) and looks super super tiny on my smartphone.